.changeset/tough-toes-allow.md

```diff
@@ -0,0 +1,6 @@
+---
+'@monite/sdk-react': minor
+'@monite/sdk-api': minor
+---
+
+add possibility to create/modify permissions for the roles
```

Comment:
Can I ask to change the description [according to multiline comments](https://www.notion.so/monite/Messages-for-Commits-MRs-Changelogs-and-Tooling-26ef378fd10141ca911ffdd4cfa6c45d?pvs=4#40cdfbcdd56948c1abe8d1572c27df68)?

Reply:
fixed